green/output.py

```diff
@@ -147,23 +154,23 @@ def __init__(
         # Susceptible to false-positives if other matching lines are output,
         # so set this to None immediately before running a coverage report to
         # guarantee accuracy.
-        self.coverage_percent = None
+        self.coverage_percent: int | None = None
 
-    def flush(self):
+    def flush(self) -> None:
         self.stream.flush()
 
-    def writeln(self, text=""):
+    def writeln(self, text: str = "") -> None:
         self.write(text + "\n")
 
-    def write(self, text):
-        if type(text) == bytes:
+    def write(self, text: str) -> None:
+        if isinstance(text, bytes):
             text = text.decode("utf-8")
         # Compensate for windows' anti-social unicode behavior
         if self._ascii_only_output and not self.disable_unidecode:
             # Windows doesn't actually want unicode, so we get
             # the closest ASCII equivalent
             text = text_type(unidecode(text))
-        # Since coverage doesn't like us switching out it's stream to run extra
```

Comment:
I always thought the english language was really off base when it decided `it's` was an exception to the apostrophe rules.

Reply:
Seriously. English is just messed up. I know the rule, and still I mess it up when I'm not thinking about it.